### PR TITLE
IPCs are Now Ignored by Slimes

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Slimes now ignore IPCs as they arent full of delicious meat to consume horribly.

## Why It's Good For The Game

Using a big-ass slime as a body shield to prevent your hitbox from being hit is gross. IPCs will be ignored by slimes.

## Testing

Abused IPCs with (emotional neglect from) slimes until horizontal.

## Changelog
:cl:
tweak: IPCs are now ignored by slimes.
/:cl:

### CODE REVIEW

**Comment by lewcc** [comment]
Do you maybe want to add some flavor text to imply what's happening? Something like "The slime's electricity conducts through your frame!"

**Comment by matttheficus** [comment]
> Do you maybe want to add some flavor text to imply what's happening? Something like "The slime's electricity conducts through your frame!"

As in only for the IPC? At the moment there is already a to_chat on `electrocute_act()`.

**Comment by roarbark** [comment]
needed change. IPCs using slimes as sprite shields to get clicked on less was really bad.

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -189,6 +189,8 @@
 
 	if(iscarbon(M))
 		var/mob/living/carbon/C = M
+		if(ismachineperson(C) && prob(10))
+			C.electrocute_act(20, src)
```
Comment (hal9000PR) [inline code/modules/mob/living/simple_animal/slime/life.dm:193]: what if they are wearing insuls? then surely the shield still works?

**Comment by Carthusia** [comment]
Huh, i thought i was noticing IPCs wearing slimes in combat

**Comment by A-Medster** [comment]
It'll be sad to see this go, even if it was never intentional. Kinda liked chilling with slimes on me, also somewhat protected others because if the slime was "eating" me than it couldnt try and hurt others.

**Comment by YoureNotTheSharpest** [comment]
not like my opinion matters but, why dont we make it so slimes ignore IPCs?

**Review by hal9000PR — APPROVED** [review]

**Review by farie82 — APPROVED** [review]
Slimes and IPCs are now confirmed allies. Oh the humanity

**Pull request merged**